### ISSUE TITLE
Idea: community example page or section

### ISSUE THREAD

**Issue by sehilyi** [gosling-lang/gosling-website#38]
When we are able to collect multiple community examples, we can add a page or a section that lists up a number of good examples generated by the community.

Similar to https://vega.github.io/vega-lite/examples/#community-examples

**Comment by wangqianwen0418** [comment]
Yes, and we can use this issue to document the community examples for now!
